_scripts/build.js

```diff
@@ -62,13 +62,6 @@ const config = {
     './dist/**/*',
     '!dist/web/*',
     '!node_modules/**/*',
-
-    // renderer
-    'node_modules/{miniget,ytsr}/**/*',
```

Comment:
I think miniget is still a ytdl-core dependency, would that cause any issues here?

Reply:
That shouldn't be a problem, ytdl-core and it's dependencies get bundled into the `renderer.js` file. Webpack leaves any dependency marked as external untouched or replaces it with your replacement. So with ytsr, the `renderer.js` file still had a `require('ytsr')` call in it, even after bundling, which also means that any dependencies of ytsr will need to be available in the `node_modules` folder at runtime, hence the miniget dependency needing to be listed here while we were using ytsr.
That also meant that we were shipping miniget twice, once inside the `node_modules` and in the `renderer.js` file.

TL;DR ytdl-core will still work as miniget is also bundled into the renderer.js file when webpack bundles ytdl-core.